### ISSUE TITLE
5841425E - Joy Ride Turbo

### ISSUE THREAD

**Issue by ZolaKluke** [xenia-project/game-compatibility#1424]
[Marketplace](https://marketplace.xbox.com/en-AU/Product/Joy-Ride-Turbo/66acd000-77fe-1000-9115-d8025841125e)

Tested on https://github.com/xenia-project/xenia/commit/f748e5af49062fd78d098acdaf045183f29faf36

# Issues:
The game performs perfectly and plays with no gameplay issues or glitches, I was able to complete the first 3 championships with no issues, however there is a weird issue where everything looks like has been censored and lowercase letters glitch on the main menu sometimes.

# Log:
[xenia.zip](https://github.com/xenia-project/game-compatibility/files/3614896/xenia.zip)

# Screenshot(s):
![image](https://user-images.githubusercontent.com/6497775/64937064-13313300-d89c-11e9-97b7-7c17d9c16928.png)
![image](https://user-images.githubusercontent.com/6497775/64937065-19bfaa80-d89c-11e9-81aa-366f067b1e2a.png)
![image](https://user-images.githubusercontent.com/6497775/64937078-293ef380-d89c-11e9-9007-932f067aaa95.png)
![image](https://user-images.githubusercontent.com/6497775/64937097-39ef6980-d89c-11e9-9b5a-18e86504a350.png)
![image](https://user-images.githubusercontent.com/6497775/64937117-525f8400-d89c-11e9-9576-6ad0f1596dfb.png)

# Labels:
state-playable gpu-corrupt-graphics

**Comment by Gliniak** [comment]
###  BootSwitch's comment from closed issue #1602 

Tested on https://github.com/xenia-project/xenia/commit/9418f3f2c5fb3b98421cca7f71506bc02fa48735

# Issues:
- Sometimes the text can get buggy and corrupted (can be fixed by pressing F5).
- The game renders at a super low resolution (the GUI is fine, just the actual graphics in game).

# Log:
[xenia.zip](https://github.com/xenia-project/game-compatibility/files/4959169/xenia.zip)

# Screenshots:
![image](https://user-images.githubusercontent.com/153369/88161187-f5e19980-cc0f-11ea-822a-7e887734d271.png)
![image](https://user-images.githubusercontent.com/153369/88161219-00039800-cc10-11ea-8fea-3c133662e8a3.png)
![image](https://user-images.githubusercontent.com/153369/88161251-098d0000-cc10-11ea-8774-87e170276d47.png)


# Labels:
gpu-drawing-corrupt state-gameplay

**Comment by Thesola10** [comment]
I don't know if it's a Wine issue, but currently on https://github.com/xenia-project/xenia/commit/f357f26eaef52c6d4c9aa69a9f069c00139da9fb the emulator crashes at the point where, on a real console, it would ask for a storage device. The screen dims and loading animation speeds up, then dies.

The `content` folder is not created.

**Comment by michaelmanzo9** [comment]
Someone please fix the graphics on the game, just looks pixelated

**Comment by Rocky331** [comment]
Hi, where did you get the image? I am searching for it sice 2020!

**Comment by LUISMAGAIVER341** [comment]
BRO WHERE YOU GOT THE ROM PLSSS

**Comment by TonyHendrickson** [comment]
Hello please someone respond to this. How can I get the full game? I only have the demo of this on PC and I have looked everywhere. Please, if you have the full game send it to me

**Comment by michaelmanzo9** [comment]
game is a rom and you need xenia to run it. very buggy, graphics are horrible.

**Comment by TonyHendrickson** [comment]
I see. I have xenia already, just don't have the full game which is what I want. I really want to play it.

**Comment by TonyHendrickson** [comment]
@michaelmanzo9

**Comment by michaelmanzo9** [comment]
Have you tried this one? https://romsfun.com/roms/xbox-360/joy-ride-turbo.html

**Comment by TonyHendrickson** [comment]
I think I did. do you have the full game when you launch on xenia??

**Comment by TonyHendrickson** [comment]
I will redownload and try again

**Comment by TonyHendrickson** [comment]
![Image](https://github.com/user-attachments/assets/d88c32e1-01ba-4f68-8dc7-542362041cfb)

**Comment by TonyHendrickson** [comment]
downloading it now

**Comment by TonyHendrickson** [comment]
ok one more minute

**Comment by TonyHendrickson** [comment]
do you think I should redownload Xenia? since i redownloaded joy ride turbo

**Comment by TonyHendrickson** [comment]
![Image](https://github.com/user-attachments/assets/3c5699da-8382-4790-82e5-af3189e17276)

**Comment by michaelmanzo9** [comment]
no, personally i think the game is too unpopular for anyone to develop anything for it. unfortunately if you want to play it properly i think your gonna have to use an xbox. i’ve been down this rabbit hole a few times

**Comment by TonyHendrickson** [comment]
I see. I've had an xbox before but I can't buy one. I just want to try it on the PC and if it is bad I won't talk about this ever again. I never got to play the full game so it would be nice to do that

**Comment by TonyHendrickson** [comment]
![Image](https://github.com/user-attachments/assets/02917a6c-53dd-4d35-bcbb-113c17f373bb) 

Well its still telling me to purchase the full game. I've lost hope now

**Comment by TonyHendrickson** [comment]
well @michaelmanzo9 I guess you are right. Thanks anyway

**Comment by Margen67** [comment]
Xenia Canary issue: https://github.com/xenia-canary/game-compatibility/issues/595
